mount/mount_windows.go

```diff
@@ -178,22 +177,10 @@ func (mounter *Mounter) List() ([]MountPoint, error) {
 
 // IsLikelyNotMountPoint determines if a directory is not a mountpoint.
 func (mounter *Mounter) IsLikelyNotMountPoint(file string) (bool, error) {
-	stat, err := os.Lstat(file)
+	_, err := os.Lstat(file)
 	if err != nil {
 		return true, err
 	}
-	// If current file is a symlink, then it is a mountpoint.
-	if stat.Mode()&os.ModeSymlink != 0 {
```

Comment:
should we keep this line? 
For linux, it checks whether it is a mount point so that it can decide either
1. unmount first if it is a mount point, otherwise
2. remove dir directly if it is not a mount point.
For windows, currently we only need one step to unmount which is delete the dir. So I think it is ok to remove this check too. Any thoughts about potential issue here?

Reply:
@jingxu97 
write like this?
```
...
if stat.Mode()&os.ModeSymlink != 0 {
   return true, err
}
return true, err
```

it's identical to 
```
return true, err
```

Reply:
enn, if we follow the the meaning of IsLikelyNotMountPoint, then it should like the following. But since for windows, current mount setup just create a symlink, and mount tear down just need to delete a symlink, the checking is actually not very necessary. As we discussed offline, as the first step of fixing this while issue, we can just make this small in this function. 

if stat.Mode()&os.ModeSymlink != 0 {
   return true, err
}
return false, err

Reply:
done, and I added more logging

Reply:
sorry I think I made it wrong, should it be 

// If current file is a symlink, then it is a mountpoint.
if stat.Mode()&os.ModeSymlink != 0 {

    return false, err
}
// it is not a mount point
return true, err

Reply:
correct, only symlink is a mount point, done